src/microbe_stage/ProcessPanel.cs

```diff
@@ -73,6 +64,6 @@ protected override void Dispose(bool disposing)
     private void OnClosePressed()
     {
         GUICommon.Instance.PlayButtonPressSound();
-        CustomHide();
+        Close();
```

Comment:
This kind of small method seems pretty common, maybe it would make sense for the base window to have a method that can be used as the button signal target? That way these couple of lines don't need to be duplicated in a bunch of derived classes.

Reply:
Done.